CMakeLists.txt

```diff
@@ -509,10 +509,20 @@ install (DIRECTORY ui/icons
   )
 include (InstallSymbolicLink)
 install_symlink (../../../../astroid/ui/icons/icon_color.png
-  ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/512x512/apps/astroid.png
+  astroid.png
```

Comment:
This installs the symlink in the current directory (likely your build tree). Same goes for line 515.